backend/tests/v5/scripts/teamspaces/createTeamspace.test.js

```diff
@@ -23,19 +23,45 @@ const {
 	generateUserCredentials,
 } = require('../../helper/services');
 
-const { src, utilScripts } = require('../../helper/path');
+const { createAccount } = require('../../helper/fronteggMock');
+
+const { src, srcV4, utilScripts } = require('../../helper/path');
 
 const CreateTeamspace = require(`${utilScripts}/teamspaces/createTeamspace`);
 
 const { disconnect } = require(`${src}/handler/db`);
-const { templates } = require(`${src}/utils/responseCodes`);
+
+jest.mock('../../../../src/v4/models/userBilling');
+const UserBillingMock = require(`${srcV4}/models/userBilling`);
 
 const user = generateUserCredentials();
+const emailUser = generateUserCredentials();
+emailUser.user = 'emailUser';
+emailUser.basicData.email = 'test@email.com';
 const teamspace = generateRandomString();
+const existingAccount = generateRandomString();
+let existingAccountId;
+
+jest.mock('../../../../src/v5/models/users', () => {
+	const original = jest.requireActual('../../../../src/v5/models/users');
+	return {
+		...original,
+		getUserByUsernameOrEmail: jest.fn((mockUser) => {
+			if (mockUser === 'problemUser') throw new Error('Made up error!');
+			if (mockUser === 'nonExistantUser') throw new Error('User not found.');
+			if (mockUser === 'test@email.com') return { user: 'emailUser' };
+
+			return { user: mockUser };
+		}),
+	};
+});
+UserBillingMock.getSubscriptionLimits.mockResolvedValue({ collaboratorLimit: 'unlimited' });
```

Comment:
This will never work. Since a newly created teamspace would not have a quota assigned to it.
![image](https://github.com/user-attachments/assets/fc205c83-d552-4d67-92a6-8d5563eafd72)

We need a bypass in the invite function to not check for quota